app/src/main/res/values/strings.xml

```diff
@@ -999,8 +999,9 @@ A level counts as a roof level when its windows are in the roof. Subsequently, r
 
     <string name="quest_crossing_title2">Is there a crossing here?</string>
     <string name="quest_crossing_yes">Yes (e.g. the curb is lowered here or there are markings)</string>
-    <string name="quest_crossing_no">No, but crossing is possible</string>
+    <string name="quest_crossing_no">No, but there is an informal crossing (e.g. informal trail)</string>
```

Comment:
The wording seems redundant to me (two times "informal").

"Informal crossing" is not a common term and the "OSM meaning" of that term is, well, OSM specific. E.g. the crossing as shown in the title image of this youtube video (found in one of the first results of a duckduckgo search for "informal crossing") https://www.youtube.com/watch?v=ctNrjp_m4To is **NOT** an `crossing=informal`, but a normal crossing (with no markings).

So, I want to avoid this term.

"Informal trail" on the other hand is better known. But not all `crossing=informal` are with informal paths. So, the hint text should reflect that, i.e. not contain "usually looks like..."

So, I'd suggest:

quest_crossing_no="No, but crossing is possible (e.g. at informal trail)"
quest_crossing_hint="A crossing with an informal path:"

Reply:
Good point. I'd also like to change the wording "but crossing is possible" a bit more towards what https://wiki.openstreetmap.org/wiki/Tag:crossing%3Dinformal says.

What about changing "informal crossing" to "intended crossing" like this:

quest_crossing_no="No intended crossing, but pedestrians commonly cross here (e.g. informal trails)"
quest_crossing_hint="An informal trail crossing a road:"

Reply:
I don't know... it's a lot of text and I don't really get what it is supposed to clarify. You know that the wiki article was mostly written by us, right?

I don't quite get what is the issue with the current wording, what exactly confused you?

Reply:
I am new to the OSM world and the first time I used StreetComplete, there was a situation where I was unsure which answer to choose for "Is there a crossing here?". Here is the pic of the crossing I took:

![Screenshot_20241002-232610](https://github.com/user-attachments/assets/d127d948-7f4e-4711-9449-a5f2b2b3ecde)

You can see that the sidewalk looks exactly the same as the road (there are no markings, no lowered curbs, no traffic signs). I wondered if this constitutes a crossing or not. Now after doing some research I think this is highway=crossing with crossing=unmarked. But as a newbie I didn't know that. Reading the answers in the app quiz I wasn't sure if I should pick "Yes (e.g. the curb is lowered here or there are markings)" or "No, but crossing is possible". I mean the curb is not lowered and there are no markings, but crossing also is possible.

Looking back I think two things would have helped me:

* An example of crossing=informal

* Being more explicit that crossings are also crossings if they do not have any markings, as long as they are intended by the municipality.

  That's why I am proposing to add the "intended crossing" or "informal crossing" wording somewhere.
  Maybe a better approach would be to add the picture from above with a text like "Crossings don't need any markings:"?

Does this make sense? Do you have any ideas on making this situation of a pedestrian crossing more clear? Or do you want to only incorporate the crossing=informal example?

Reply:
Ah, thank you for this explanation. Well, as far as I know, for situations like above - da scheiden sich die Geister - there is no consensus in the community as far as I am aware.

Some argue, there shouldn't be a `highway=crossing` at all, others argue that the "Road" in red in your example, even if just some sort of alley, could just as well be asphalted and lowered, in which case it is definitely a `highway=crossing`, so tagging it always is fair.

Reviewing what I remember of the discussion(s), I tend to agree with the latter group, although while it is not clear in the wiki documentation, it would be problematic to propagate one over the other. So, this is the reason for why the example in your picture is not really mentioned at all in StreetComplete.

It is also the reason why StreetComplete never asks this quest for intersections of footways with `highway=service` in order to avoid asking users at all for situations like pictured above. Isn't the above a driveway of sorts? So, I am puzzled why you were asked at all.
Anyway, I guess `highway=residential` roads that look like above also exist, although rarely.

I understand the issue better now and your suggestions make sense. Maybe using "intended" in wording indeed helps making it clearer:

> **Is there a crossing here?**
> 
> - Yes, intended as crossing
> - Not intended as crossing, still possible to cross (e.g. informal trails)
> - No, crossing is prohibited or impossible

I gather that including the examples in the "yes" answer may actually rather confuse than help, as no markings are necessary for something to be a crossing. And as mapping sidewalks as separate ways instead of properties of the roadway is becoming more and more popular, we have more and more cases where there are just "implicit" crossings on every intersection, even if it is deep in a residential area (with no markings, traffic signals etc). And sometimes, the curbs are just not lowered there.

Reply:
Thanks for your detailed answer :)

> Isn't the above a driveway of sorts? So, I am puzzled why you were asked at all.

I will go and check sometime this week. I don't remember.